src/main/java/com/openlattice/shuttle/transformations/TransformValueMapper.java

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
Maaaaybe Ez Kotlinify though it's used in a lot of places so potential for terror

Reply:
😱